src/elements/fields/SignatureField.jsx

```diff
@@ -10,9 +9,15 @@ function SignatureField({
   signatureRef = {},
   elementProps = {},
   onEnd,
+  onClear,
+  isClearInitiallyVisible,
```

Comment:
defaultClearVisibility?

Reply:
also should it = false in the parameters

Reply:
I'm not sure if default correctly conveys the purpose of this prop

I wrote a comment in <Form /> (https://github.com/feathery-org/feathery-react/pull/365/files#diff-421a55da4cd6f6a06fadba2ad8f542f24b877f82de1dbdfd9271e482d69b91edR1404) about why this prop exists - so that we know whether the initialState should be true/false based on whether or not there's an existing session value or if it's set with `initialValues` 

It seemed more appropriate to put the comment in Form because this prop is required due to the implementation details of Form, not SignatureField